packages/SwingSet/test/stores/test-collections.js

```diff
@@ -408,6 +408,35 @@ test('set fail on concurrent modification', t => {
   }, m(`keys in store cannot be added to during iteration`));
 });
 
+test('fail on oversized keys', t => {
```

Comment:
Let's add at least one knife-edge test: assert that exactly 219 chars passes, but exactly 220 chars fails. If we're making this part of the observable/consensus behavior, we should make it clear that it can't be changed without carefully considering the compatibility consequences.